suite/tests/api/static_sideline.expect

```diff
@@ -1,17 +1,17 @@
 pre-DR init
 in dr_client_main
+client thread 0 is alive
+client thread 1 is alive
+client thread 2 is alive
+client thread 3 is alive
 pre-DR start
-client thread is alive
-client thread is alive
-client thread is alive
-client thread is alive
 Saw some bb events
 post-DR detach
 re-attach attempt
 in dr_client_main
-client thread is alive
-client thread is alive
-client thread is alive
-client thread is alive
+client thread 0 is alive
+client thread 1 is alive
+client thread 2 is alive
+client thread 3 is alive
 Saw some bb events
 all done
```

Comment:
I saw 6 out of 200 runs fail with a missing "all done" line or something.  Does yours pass at least 200 runs in a row?  Probably 2000 is a better test.